hack/tools/install_yq.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
do we need it ?

Reply:
it is needed for kubectl installation. it should be added a dependency to it. please look at install_kubebuilder.sh